src/main/java/org/gridsuite/modification/server/entities/elementary/EquipmentCreationEntity.java

```diff
@@ -0,0 +1,38 @@
+/*
+  Copyright (c) 2021, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.entities.elementary;
```

Comment:
Create new packages `org.gridsuite.modification.server.entities.equipment.creation `and `org.gridsuite.modification.server.entities.equipment.attribute.modification` ?

Reply:
Done